digits/model/tasks/caffe_train.py

```diff
@@ -1490,3 +1501,38 @@ def get_network_desc(self):
         return text description of model
         """
         return text_format.MessageToString(self.network)
+
+    def net_sanity_check(self, net, phase):
```

Comment:
Sure, or perhaps a `staticmethod` since we're not making any reference to the class?
And I need to write tests too, thanks for reminding me :-)